barretenberg/cpp/src/barretenberg/vm2/simulation/execution.test.cpp

```diff
@@ -100,6 +100,8 @@ TEST_F(ExecutionSimulationTest, Call)
     EXPECT_CALL(context, get_next_pc);
     EXPECT_CALL(context, get_is_static);
     EXPECT_CALL(context, get_msg_sender).WillOnce(ReturnRef(parent_address));
+    FF zero = FF(0);
```

Comment:
Maybe move this together with the other variables, so that it can be reused.

Also `FF zero = 0` should work.